src/main/resources/db/migration/V053__Add_ccd_id_and_action_to_envelopes.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE envelopes
+  ADD COLUMN ccdId VARCHAR(50) NULL,
```

Comment:
If the value is long, shouldn't the column type be bigint?

Reply:
that is the question, Leszek recommended it and I agreed it might be better.

Reply:
it's an external ID, safer to have it as varchar